.github/workflows/buildsCI.yaml

```diff
@@ -585,6 +585,41 @@ jobs:
         cmake .. -DCMAKE_BUILD_TYPE=${{env.LRS_BUILD_CONFIG}} -DBUILD_EXAMPLES=true -DBUILD_WITH_OPENMP=false -DHWM_OVER_XU=false -DCHECK_FOR_UPDATES=true
         cmake --build . -- -j4
         ls
+        
+   #--------------------------------------------------------------------------------
+  
+  Mac_DDS_cpp:
+    runs-on: macos-15
+    timeout-minutes: 60
+    
+    steps:
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 #v4
+
+    - name: Check_API
```

Comment:
We can skip it IMO, we have it on other steps that will fail if needed

Reply:
ok